Dockerfile

```diff
@@ -1,8 +1,30 @@
-# Parent Image
-FROM ubuntu:latest
+# RMG Dockerfile 
+# The parent image is the base image that the Dockerfile builds upon.
+# The RMG installation instructions suggest Anaconda for installation by source, however, we use micromamba for the Docker image due to its smaller size and less overhead.
+# https://hub.docker.com/layers/mambaorg/micromamba/1.4.3-jammy/images/sha256-0c7c97be938c5522dcb9e1737bfa4499c53f6cf9e32e53897607a57ba8b148d5?context=explore
+# We are using the sha256 hash to ensure that the image is not updated without our knowledge. It considered best practice to use the sha256 hash
+FROM --platform=linux/amd64 mambaorg/micromamba@sha256:0c7c97be938c5522dcb9e1737bfa4499c53f6cf9e32e53897607a57ba8b148d5
 
-# Install Bash shell
-RUN ln -snf /bin/bash /bin/sh
+# Set the user as root
+USER root
```

Comment:
Sorry wasn't meant to press resolve. Fat fingers on the phone. So I specify root because if I recall, that's how they suggest building when using a micromamba base